<ui version="4.0" >
 <class>udg::QThickSlabWidgetBase</class>
 <widget class="QWidget" name="udg::QThickSlabWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>65</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Mode: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="m_projectionModeComboBox" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="m_slabThicknessLabel" >
       <property name="text" >
        <string>2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="m_slabThicknessSlider" >
       <property name="minimum" >
        <number>2</number>
       </property>
       <property name="maximum" >
        <number>100</number>
       </property>
       <property name="pageStep" >
        <number>1</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::NoTicks</enum>
       </property>
       <property name="tickInterval" >
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
